components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/Utils.java

```diff
@@ -106,4 +109,19 @@ public static ServiceAdmin getServiceAdmin(MessageContext messageContext) {
         }
         return serviceAdmin;
     }
+
+    /**
+     * Method to update pax-logging configuration.
+     */
+    public static void updateLoggingConfiguration() throws IOException {
+
+        ConfigurationAdmin configurationAdmin = ConfigurationHolder.getInstance().getConfigAdminService();
+        if (configurationAdmin != null) {
```

Comment:
Can this ever be null? ServiceBusInitializer will not get activated without this.

Reply:
Fixed